### ISSUE TITLE
"zed manage -monitor" exits if a "-pool" target doesn't exist

### ISSUE THREAD

**Issue by philrz** [brimdata/zed#5266]
## tl;dr

When running `zed manage -monitor`, if one of the targets specified in a `-pool` option doesn't exist, the command exits. Since this is designed to be a long-running process, this seems undesirable.

## Details

Repro is with Zed commit 038a6ef.

Check out this attached video where I'm trying to use `-pool` to selectively manage a subset of my pools. In this case I made a mistake in one of my pool names in `-pool` but this did not become apparent until the end of the first interval, at which time `zed manage` exited.

https://github.com/user-attachments/assets/df15c9d3-3f18-4ca7-ad69-b9840a3186cb

Of course, in this case I lowered the interval to 5 seconds to make the video watchable. In practice the default interval is one minute, and operationally users may end up setting this to an hour or more.

This highlights a quirk of `zed manage` I've always been aware of but haven't previously surfaced: Having to wait to the end of the first interval to see an update is a little odd, and this case feels kinda bad because I may have spun up the command on a host, saw it start and thought it's going to run continuously, but now it's not running at all.

Here's another variation where all the pools specified exist such that the update runs ok at the first interval, but then if one of those pools gets dropped, once again `zed manage` exits at the next interval.

https://github.com/user-attachments/assets/bbcdda3e-e739-4746-b8e1-57813d12b124

This feels even more hazardous since one user's maintenance task of dropping a single pool can tank another user's maintenance task that was potentially affecting many pools (e.g., compaction, creating vectors, etc.)
